zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -325,6 +328,13 @@ public static Either<ProblemDetail, UserDTO> toUserDTO(
                 passwordEncoder.encode(request.getPassword())));
   }
 
+  public static Either<ProblemDetail, MappingDTO> toMappingRequest(
```

Comment:
```suggestion
  public static Either<ProblemDetail, MappingDTO> toMappingDTO(
```
for consistency with toUserDTO().

Reply:
We should create a follow-up also for this case, we have a mix of `DTO` and `request` object. Can you do this ?